src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1988,6 +1988,55 @@ softwareA,softwareB
 
 As expected, if 10 replicated tasks requiring software_A,software_B licenses are submitted, they will be executed 3 by 3.
 
+=== Earliest deadline first policy
+
+Scheduler has `Earliest deadline first policy` (EDF policy) that priorities jobs with deadlines.
+
+The EDF Policy considers two Generic Info variables: `JOB_DDL` and `JOB_EXEC_TIME`.
```

Comment:
I think that you should remove the word "variables". It could be confusing between the "Generic Info" variables and the "Variables" variables.

Reply:
You are right